content/docs/render-props.md

```diff
@@ -209,17 +209,17 @@ class MouseTracker extends React.Component {
 }
 ```
 
-Now, instead of effectively cloning the `<Mouse>` component and hard-coding something else in its `render` method to solve for a specific use case, we provide a `render` prop that `<Mouse>` can use to dynamically determine what it renders.
+Ahora, en lugar de clonar efectivamente el componente `<Mouse>` y codificar forma fija otra cosa en su método `render` para resolver un caso de uso específico, proporcionamos una `render prop` que `<Mouse>` pueda usar para dinámicamente determinar que renderizar.
```

Comment:
I'd put `y codificar algo más en su...` instead of `y codificar forma fija otra cosa en su`.

Reply:
```suggestion
Ahora, en lugar de clonar efectivamente el componente `<Mouse>` y codificar de forma fija otra cosa en su método `render` para resolver un caso de uso específico, proporcionamos una `render prop` que `<Mouse>` pueda usar para dinámicamente determinar que renderizar.
```